docs/Release-Process.md

```diff
@@ -46,12 +46,8 @@ and trustee versions were updated when their components released as listed above
 As the [CoCo operator](https://github.com/confidential-containers/operator/) doesn't release until after peer pods,
 the [current plan](https://github.com/confidential-containers/confidential-containers/pull/201#discussion_r1570606331),
 is to pick the latest operator commit to pin that in our released version's instructions of deploying the operator.
-To do this, we should update the `git.coco-operator.reference` value in [versions.yaml](../src/cloud-api-adaptor/versions.yaml).
-
-When this change is merged, it triggers the
-[project images publish workflow](../.github/workflows/publish_images_on_push.yaml) to create a new container image in
-[`quay.io/confidential-containers/cloud-api-adaptor`](https://quay.io/repository/confidential-containers/cloud-api-adaptor?tab=tags)
-to use in testing.
+To do this, we should update the `git.coco-operator.reference` value in [versions.yaml](../src/cloud-api-adaptor/versions.yaml)
+and create a commit for this and add the following changes as a second commit to the same PR.
```

Comment:
By `the following changes` are you referring to all the changes mentioned in the next section ?

Reply:
Yeah, perhaps it's clear, but in the alpha1 I followed the instructions to create two separate PRs for this, but there is no dependency, so wanted to skip a review cycle